### ISSUE TITLE
Errors when installing with pip3 install 

### ISSUE THREAD

**Issue by 0xJeti** [honoki/bbrf-client#33]
Hi Honoki,

I was trying to install fresh copy of bbrf-client using new method using `pip3 install bbrf` (my VPS has both python2 and python3 installed).

But after installation I'm getting this error:
```
$ pip3 install bbrf
Collecting bbrf
  Cache entry deserialization failed, entry ignored
  Downloading https://files.pythonhosted.org/packages/b6/76/5b258579f087be9b418f6bf63400ccdae1cd34bb4680bf26e701601ccb27/bbrf-1.0.7-py3-none-any.whl
Collecting slackclient==1.3.2 (from bbrf)
  Cache entry deserialization failed, entry ignored
Collecting docopt (from bbrf)
  Cache entry deserialization failed, entry ignored
[...]
Installing collected packages: chardet, certifi, urllib3, idna, requests, six, websocket-client, slackclient, docopt, bbrf
Successfully installed bbrf-1.0.7 certifi-2020.12.5 chardet-4.0.0 docopt-0.6.2 idna-2.10 requests-2.25.1 six-1.15.0 slackclient-1.3.2 urllib3-1.26.3 websocket-client-0.54.0

$ bbrf
Traceback (most recent call last):
  File "/home/xxx/.local/bin/bbrf", line 7, in <module>
    from bbrf import main
ModuleNotFoundError: No module named 'bbrf'
```

**Comment by honoki** [comment]
Hi @0xJeti - thanks for letting me know. I thought I removed the 1.0.7 release from pip because of errors. Could you try again by forcing 1.0.6 as follows:

```
pip3 install bbrf==1.0.6
```

**Comment by 0xJeti** [comment]
Yes, that works (unfortunately without 1.0.7 cool features) ;)

**Comment by honoki** [comment]
My bad! I'm new to python packaging - will make sure the next version fixes that!

**Comment by honoki** [comment]
Should all be fixed in 1.0.8!
